src/routes/v2/authenticated/sites.ts

```diff
@@ -84,6 +84,25 @@ export class SitesRouter {
     return res.status(200).json({ stagingUrl })
   }
 
+  getSiteInfo: RequestHandler<
+    { siteName: string },
+    unknown,
+    never,
+    never,
+    { userSessionData: UserWithSiteSessionData }
+  > = async (req, res) => {
+    const { userSessionData } = res.locals
+    const { siteName } = req.params
+    const userWithSiteSessionData = this.addSiteNameToSessionData(
+      userSessionData,
+      siteName
+    )
```

Comment:
Actually we should already have siteName in the sessionData here! Via the extra attachSiteHandler middleware for this route

Reply:
Removed in a12ad20!